xml/System.Net/Cookie.xml

```diff
@@ -968,7 +968,8 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Net.Cookie.Value%2A> of a <xref:System.Net.Cookie> must not be `null`. The following characters are reserved and cannot be used for this property: semicolon, comma.  
+ The <xref:System.Net.Cookie.Value%2A> of a <xref:System.Net.Cookie> must not be `null`. If a `null` value is assigned to this property, it is replaced with an empty string. 
+ The following characters are reserved and cannot be used for this property: semicolon, comma. Assigning invalid characters to this property does not throw an exception, but when you try to add this <xref:System.Net.Cookie> to a <xref:System.Net.CookieContainer> instance with the <xref:System.Net.CookieContainer.Add%2A> method, the operation will fail and throw an exception.
```

Comment:
> The following characters are reserved and cannot be used for this property: semicolon, comma.

With only two, here's a suggestion that reads a bit more naturally ...

> Semicolons and commas are reserved characters that cannot be used in the value of this property.

Few other little nits in there ...
* "does not" :point_right: "doesn't"
* Semicolon before "but"

I also have a passing question based on https://msdn.microsoft.com/library/kesbe27x(v=vs.110).aspx regarding escaping with a double quote. Should "... used in the value of this property" be extended to say "... used in the value of this property unless they're escaped"? If so, it requires adding in the "how" bit on using the double quote to do it.